deploy/helm/templates/statefulset.yaml

```
@@ -46,12 +46,12 @@ spec:
       initContainers:
       {{- if .Values.redis.enabled }}
       - name: redis-init-container
-        image: alpine
+        image: {{ .Values.redis.initImages | default "alpine" }}
```

Can we use the singular `image`, to make the meaning obvious please?

```suggestion
        image: {{ .Values.redis.image | default "alpine" }}
```

---

I see your point, unfortunately the key **image** is passed [down to the subcharts of mongo](https://github.com/bitnami/charts/tree/b95989761f3feccc073f6739d4df62bf59ae34b7/bitnami/mongodb#mongodb-parameters) and [redis](https://github.com/bitnami/charts/tree/b95989761f3feccc073f6739d4df62bf59ae34b7/bitnami/redis#redis-image-parameters). 

Using it this way results in a parsing error:
```
$ git --no-pager diff
diff --git a/deploy/helm/templates/statefulset.yaml b/deploy/helm/templates/statefulset.yaml
index e42fd091d..b76251d71 100644
--- a/deploy/helm/templates/statefulset.yaml
+++ b/deploy/helm/templates/statefulset.yaml
@@ -51,7 +51,7 @@ spec:
       {{- end }}
       {{- if .Values.mongodb.enabled }}
       - name: mongo-init-container
-        image: {{ .Values.mongodb.initImages | default "docker.io/bitnami/mongodb:4.4.11-debian-10-r12" }}
+        image: {{ .Values.mongodb.image | default "docker.io/bitnami/mongodb:4.4.11-debian-10-r12" }}
         command: ['sh', '-c', "until mongo --host appsmith-mongodb.{{.Release.Namespace}}.svc.cluster.local --eval 'db.runCommand({ping:1})' ; do echo waiting for mongo; sleep 2; done"]
       {{- end }}
       containers:
diff --git a/deploy/helm/values.yaml b/deploy/helm/values.yaml
index f912aaeb0..6b8453374 100644
--- a/deploy/helm/values.yaml
+++ b/deploy/helm/values.yaml
@@ -9,7 +9,7 @@ redis:
 
 mongodb:
   enabled: true
-  initImages: docker.io/bitnami/mongodb:4.4.11-debian-10-r12
+  image: REGISTRY.LOCAL/bitnami/mongodb:4.4.11-debian-10-r12
   service:
     nameOverride: appsmith-mongodb
   auth:
```

```
$ helm template -n test appsmith-init-images .                    
Error: template: appsmith/charts/mongodb/templates/replicaset/statefulset.yaml:56:10: executing "appsmith/charts/mongodb/templates/replicaset/statefulset.yaml" at <include "mongodb.imagePullSecrets" .>: error calling include: template: appsmith/charts/mongodb/templates/_helpers.tpl:87:3: executing "mongodb.imagePullSecrets" at <include "common.images.pullSecrets" (dict "images" (list .Values.image .Values.metrics.image .Values.volumePermissions.image) "global" .Values.global)>: error calling include: template: appsmith/charts/mongodb/charts/common/templates/_images.tpl:36:14: executing "common.images.pullSecrets" at <.pullSecrets>: can't evaluate field pullSecrets in type interface {}

Use --debug flag to render out invalid YAML`
```

On a second thought it might even be better to keep the initContainer values out of the redis & mongo block, to avoid the misinterpretation that they could be passed through to the subcharts, at the end they are used in the appsmith chart only. 

I'll update the PR,  let me know what you think of the alternative approach